src/components/vars.sass

```diff
@@ -178,8 +258,7 @@ $tab-min-height: 50px
   align-items: center
   text-align: $direction
   color: #474748
-  font-size: 12px
-  font-weight: 700
+  font-size: 14px
```

Comment:
This causes us to lose the previous styling on the right nav tabs.  In the Dataflow case, it definitely looks worse.